examples/AutoFilterTutorial.0.0.cpp

```diff
@@ -0,0 +1,142 @@
+// AutoFilterTutorial.0.0.cpp created by Victor Dods on 2015/06/03
+// Copyright (C) Leap Motion, Inc. All rights reserved.
```

Comment:
Standard requires that this be the first line--that's why the build server is failing right now.  Also we have a standard form for this notice, it looks like this:

```C++
// Copyright (c) 2010 - 2013 Leap Motion. All rights reserved. Proprietary and confidential.
```